src/Components/SmartComponents/Reports/ReportsPage.test.js

```diff
@@ -9,7 +9,8 @@ import { useSelector } from 'react-redux';
 /* Mock function returning permissions */
 jest.mock('../../../Helpers/Hooks', () => ({
     ...jest.requireActual('../../../Helpers/Hooks'),
-    useRbac: () => [[true, true, true], false]
+    useRbac: () => [[true, true, true], false],
+    useFeatureFlag: () => true
```

Comment:
The tests (all in this PR) should by default run with the feature turned off/`false`. and tests added for the feature should turn them on explicitly. 

This way we can ensure that the functionality stays the same until the feature is realeased.